src/copy.c

```diff
@@ -102,7 +103,8 @@ typedef struct TSCopyMultiInsertBuffer
  */
 typedef struct TSCopyMultiInsertInfo
```

Comment:
Could you add the comment here that explains why it is useful to maintain chunk-id to buffer mapping - especially since the code will no longer have the comment at line 897.

Reply:
@gayyappan Thank you for pointing this out. I improved the comment.